ni_measurementlink_service/session_management/_reservation.py

```diff
@@ -887,6 +1099,45 @@ def get_nidcpower_connections(
             nidcpower.Session, pin_names, sites, INSTRUMENT_TYPE_NI_DCPOWER
         )
 
+    def get_nidcpower_connections_with_multiplexer(
+        self,
+        multiplexer_session_type: Type[TMultiplexerSession],
+        pin_names: Union[str, Iterable[str], None] = None,
+        sites: Union[int, Iterable[int], None] = None,
+    ) -> Sequence[TypedConnectionWithMultiplexer[nidcpower.Session, TMultiplexerSession]]:
+        """Get all NI-DCPower connections matching the specified criteria.
+
+        Args:
+            multiplexer_session_type: The multiplexer(s) session type.
```

Comment:
multiplexer_session_type: The multiplexer session type.

Reply:
I included the additional `(s)` to denote that there could be multiple multiplexers but of the same session type.